dhis2/src/main/resources/webapp/messages/messages.properties

```diff
@@ -32,6 +32,8 @@ dhis2.web.settings.serverURI = Server URI
 dhis2.web.settings.username = Username
 dhis2.web.settings.password = Password
 dhis2.web.settings.save = Save
+dhis2.web.settings.save.success = Save succesfull.
```

Comment:
1. Successful  2. Can we expand this message a little bit? Eg. "Settings have been successfully saved."